sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerClient.java

```diff
@@ -584,4 +585,111 @@
+        String iDDocumentUrl) {
+        return beginRecognizeIdDocumentsFromUrl(iDDocumentUrl, null, Context.NONE);
+    }
+
+    /**
+     * Analyze ID documents using optical character recognition (OCR) and a prebuilt ID trained
+     * model to extract key information from passports.
+     * See <a href="https://docs.microsoft.com/azure/cognitive-services/form-recognizer/concept-identification-cards#fields-extracted">here</a> for fields found on an ID document.
+     * <p>The service does not support cancellation of the long running operation and returns with an
+     * error message indicating absence of cancellation support</p>
+     *
+     * <p><strong>Code sample</strong></p>
+     * {@codesnippet com.azure.ai.formrecognizer.FormRecognizerClient.beginRecognizeIDDocumentsFromUrl#string-RecognizeIDDocumentOptions-Context}
+     *
+     * @param iDDocumentUrl The source URL to the input ID Document.
+     * @param recognizeIDDocumentOptions The additional configurable {@link RecognizeIDDocumentOptions options}
+     * that may be passed when analyzing an ID Document.
+     * @param context Additional context that is passed through the HTTP pipeline during the service call.
+     *
+     * @return A {@link SyncPoller} to poll the progress of the recognize ID Document operation until it has
+     * completed, has failed, or has been cancelled. The completed operation returns a list of {@link RecognizedForm}.
+     * @throws FormRecognizerException If recognize operation fails and the {@link AnalyzeOperationResult} returned with
+     * an {@link OperationStatus#FAILED}.
+     * @throws NullPointerException If {@code iDDocumentUrl} is null.
+     */
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    public SyncPoller<FormRecognizerOperationResult, List<RecognizedForm>> beginRecognizeIdDocumentsFromUrl(
+        String iDDocumentUrl, RecognizeIDDocumentOptions recognizeIDDocumentOptions, Context context) {
+        return client.beginRecognizeIdDocumentsFromUrl(iDDocumentUrl, recognizeIDDocumentOptions, context)
+            .getSyncPoller();
+    }
+
+    /**
+     * Analyze ID documents using optical character recognition (OCR) and a prebuilt ID trained
+     * model to extract key information from passports.
+     * See <a href="https://docs.microsoft.com/azure/cognitive-services/form-recognizer/concept-identification-cards#fields-extracted">here</a> for fields found on an ID document.
+     * <p>The service does not support cancellation of the long running operation and returns with an
+     * error message indicating absence of cancellation support</p>
+     *
+     * <p><strong>Code sample</strong></p>
+     * {@codesnippet com.azure.ai.formrecognizer.FormRecognizerClient.beginRecognizeIDDocuments#InputStream-long}
+     *
+     * @param iDDocument The data of the business card to recognize ID Document information from.
```

Comment:
business card snuck in here again 😄 

edit: same with docstring below this one